active_model_serializers.gemspec

```diff
@@ -42,7 +42,8 @@ Gem::Specification.new do |spec|
   # 'minitest'
   # 'thread_safe'
 
-  spec.add_runtime_dependency 'jsonapi', '0.1.1.beta6'
+  # TODO: Latest jsonapi-renderer is 0.1.2
+  spec.add_runtime_dependency 'jsonapi-renderer', '0.1.1.beta1'
```

Comment:
ah, so that's the version in the gem. Sure, let's do this. Can you add a changelog under misc?